### ISSUE TITLE
Include QR code in ticket confirmation email

### ISSUE THREAD

**Issue by GROwen** [drupalmel/drupalcampmel#23]
Providing the QR code in the ticket/account confirmation email would help when it comes time to accessing it is users would then have the ability to refer to email offline when needing to present QR code at registration.

**Comment by Decipher** [comment]
Absolutely. Ideally there would be some sort of 'design your own lanyard' functionality in the future that would allow users to choose what elements they want on their lanyard (D.o. username, realname, job position, etc) as well as the QR code for entry.

At the bare minimum, the QR Code should be in the confirmation email as mentioned here, but the functionality wasn't built till well after the original round of emails where sent.
